server/config.go

```diff
@@ -192,6 +199,24 @@ func NewLogConfig() *LogConfig {
 	}
 }
 
+// MetricsConfig is configuration relevant to metrics capturing and output.
+type MetricsConfig struct {
+	ReportingFreqSec     int    `yaml:"reporting_freq_sec" json:"reporting_freq_sec" usage:"Frequency of metrics exports. Default is 1 second."`
```

Comment:
Is 1 sec a good default here? Feels a bit too frequent for metrics reporting to a (potentially) remote target. Maybe 5-10 sec makes more sense?